lib/presentation/pages/add_expenses_page/bloc/add_expenses_state.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:trip_tally/presentation/utils/enums/errors.dart';
+
+part 'add_expenses_state.freezed.dart';
+
+@freezed
+class AddExpensesState with _$AddExpensesState {
```

Comment:
```suggestion
class CreateExpensesState with _$CreateExpensesState {
```

everywhere use Create not Add